src/validator/CreditCardVendor.java

```diff
@@ -0,0 +1,32 @@
+package validator;
+
+import java.util.ArrayList;
+
+public class CreditCardVendor {
```

Comment:
since this is basically a DTO, let's make it immutable to save us some trouble in the future. Add `final` to class, fields and return copies of collections instead of collections themselves. Also add a private, no-arg constructor.